.editorconfig

```diff
@@ -12,6 +12,9 @@ trim_trailing_whitespace = true
 max_line_length = off
 trim_trailing_whitespace = false
 
+[*.sql]
+insert_final_newline = true
```

Comment:
We will need this for all files I guess.  Can make line 8 itself as true?

Reply:
sure thing we could apply the rule to all files, let me see what I could do